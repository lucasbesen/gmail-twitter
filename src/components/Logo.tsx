import styled from 'styled-components';

const Logo = styled.img.attrs({
  src: 'https://ssl.gstatic.com/ui/v1/icons/mail/rfr/logo_gmail_lockup_default_2x.png',
})`
  width: 109px;
  height: 40px;
`;

export default Logo;
