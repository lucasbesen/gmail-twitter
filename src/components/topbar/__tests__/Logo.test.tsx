import React from 'react';
import renderer from 'react-test-renderer';

import Logo from '../Logo';

it('should render Logo without error', () => {
  const component = renderer.create(<Logo />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
