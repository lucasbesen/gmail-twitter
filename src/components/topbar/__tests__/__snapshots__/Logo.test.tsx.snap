// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Logo without error 1`] = `
<img
  className="sc-bdVaJa llObtl"
  src="https://ssl.gstatic.com/ui/v1/icons/mail/rfr/logo_gmail_lockup_default_2x.png"
/>
`;
